app/lib/omd_rrule.rb

```diff
@@ -0,0 +1,16 @@
+class OmdRrule
+  def initialize(*args)
+    self.rule = RRule::Rule.new(*args)
+  end
+
+  def last_occurrence
+    rule.between(Time.zone.now.beginning_of_day, 3.years.from_now.end_of_day).last
```

Comment:
Jason recommended being explicit instead of using the `from_now` helper method since the app is being built around time zones. https://api.rubyonrails.org/classes/ActiveSupport/Duration.html the helper methods use `Time.now.advance` which will use the default server time zone.
```suggestion
    rule.between(Time.zone.now.beginning_of_day, Time.zone.now.advance(years: 3).end_of_day).last
```

Reply:
Perfect, they are the same.